src/lib/haptics.ts

```diff
@@ -13,26 +13,30 @@ const hapticImpact: ImpactFeedbackStyle = isIOS
   : ImpactFeedbackStyle.Light // Users said the medium impact was too strong on Android; see APP-537s
 
 export class Haptics {
-  static default() {
-    if (isWeb) {
+  static default(disabled: boolean) {
+    if (disabled || isWeb) {
       return
     }
+
     impactAsync(hapticImpact)
   }
-  static impact(type: ImpactFeedbackStyle = hapticImpact) {
-    if (isWeb) {
+  static impact(type: ImpactFeedbackStyle = hapticImpact, disabled: boolean) {
```

Comment:
Where are these being used? I can't find callsites. Are these here just in case to match the Expo API?

Reply:
I assume just to match, yea. That's how it was set up prior so I maintained the API but I'm fine with removing these.

Reply:
Simplified this down to just `playHaptic(disabled: boolean)`. I don't think we have any intentions of using these other options, those originated from a while back and it does not look like they were ever used from what I can tell.